src/electionguard/logs.py

```diff
@@ -21,8 +21,7 @@ def __init__(self) -> None:
         super(ElectionGuardLog, self).__init__()
 
         self.__logger = logging.getLogger("electionguard")
-        self.__logger.addHandler(self._get_file_handler())
```

Comment:
Note: The file handler is removed. It can now be added at any point, but is not the default.